docs/quick-tutorial.md

```diff
@@ -83,9 +83,9 @@ If you are not interested in backing up any data, the `--objstore.config-file` f
 
 ### Store API
 
-The Sidecar component implements and exposes a gRPC *[Store API](https://github.com/thanos-io/thanos/blob/main/pkg/store/storepb/rpc.proto#L27)*. The sidecar implementation allows you to query the metric data stored in Prometheus.
+The sidecar component implements and exposes a gRPC *[Store API](https://github.com/thanos-io/thanos/blob/main/pkg/store/storepb/rpc.proto#L27)*. The sidecar implementation allows you to query the metric data stored in Prometheus.
```

Comment:
```suggestion
The Sidecar component implements and exposes a gRPC *[Store API](https://github.com/thanos-io/thanos/blob/main/pkg/store/storepb/rpc.proto#L27)*. The sidecar implementation allows you to query the metric data stored in Prometheus.
```

Reply:
This may be a case where leaving `The` is okay since `Sidecar` is followed by `component`. Bit of a squishy case though, especially since it's followed by a sentence starting with `The sidecar implementation`.

There's also an uncapitalized `the sidecar` in line 65 -- worth changing?